.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Don't commit this file. Add `.vscode` to .gitignore instead

Reply:
Updated